.github/workflows/test-report.yml

```diff
@@ -0,0 +1,16 @@
+name: 'Test Report'
```

Comment:
`Test` sounds like a verb here to me.  Maybe `Generate test report`?  Same in the file name.

Reply:
Fair point! This is from the template at https://github.com/dorny/test-reporter so I opened a PR to fix it there too. :)